manual/core/statements/prepared/README.md

```diff
@@ -131,6 +131,8 @@ This can be ambiguous if the query uses the same column multiple times, like in
 where sku = ? and date > ? and date < ?`. In these situations, use positional setters or named
 parameters.
 
+#### Unset values
+
 With native protocol V3, all variables must be bound. With native protocol V4 or above, variables
```

Comment:
Would be good to link to native protocol section here

Reply:
👍 
I also updated the per-query keyspace page.